### DISCUSSION TITLE
postal version command not working

### DISCUSSION THREAD

**Discussion by LuD1161** [postalserver/postal#2710] [Feature suggestions]
## Describe the bug

The postal CLI doesn't print the version. I tried `postal version` command, but it just shows the help page again.

## To Reproduce

1. Go to terminal
2. Enter `postal version`

## Expected behaviour

The current version should be printed.

## Screenshots

<img width="804" alt="image" src="https://github.com/postalserver/postal/assets/17861054/5265c626-b7a5-45c4-8472-4dc97c031d16">

## Environment details 

 - OS: Ubuntu 22.04
 - Version: `postal 2.1.4` ( I found this by `postal upgrade` command )

**Comment by willpower232** [comment]
Yes it would be nice to have a kind of version output here https://github.com/postalserver/install/blob/main/bin/postal

You can see how the `set-version` command works but might be a bit complicated to extract the version reliably.
